Modelica_Synchronous/ClockSignals.mo

```diff
@@ -88,10 +88,18 @@ For an example, see
       parameter Integer resolutionFactor = conversionTable[Integer(resolution)] annotation(Evaluate=true);
       Clock c annotation(HideResult=true);
     equation
+      // The following then-branch using subSample corresponds to the simpler Clock(factor*resolutionFactor, 1),

```

Comment:
You mean the else branch usinf supesample.

Reply:
Hm, the comment makes sense to me. It's the "then" branch which uses the subSample construction for potentially covering a longer time duration.